src/app/components/room/room.service.js

```diff
@@ -213,7 +213,7 @@
           pluginHandle.send({"message": request, success: function(result) {
             // Free the resource (it looks safe to do it here)
             pluginHandle.detach();
-
+            console.log("This is the result", result);
```

Comment:
If you think it's useful even in production, please improve the description. If not, just remove this line.